app/src/main/java/com/bignerdranch/android/draganddraw/BoxDrawingView.kt

```diff
@@ -0,0 +1,73 @@
+package com.bignerdranch.android.draganddraw
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.PointF
+import android.util.AttributeSet
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+
+class BoxDrawingView(context: Context, attrs: AttributeSet? = null) :
+    View(context, attrs) {
+
+    private var currentBox: Box? = null
+    private val boxen = mutableListOf<Box>()
```

Comment:
was that supposed to be named `boxen` ? 😅

Reply:
Yep😄